<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainModelPage</class>
 <widget class="QWidget" name="TrainModelPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型训练</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>680</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="gboxMain">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>480</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="lay1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labVideo1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>480</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>304</width>
     <height>656</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>SVM操作区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>标签路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="svmDataDir"/>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="svmSelDataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <layout class="QFormLayout" name="formLayout_7">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>测试路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="svmTestDir"/>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="svmSelTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QFormLayout" name="formLayout_8">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>异常数据</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="svmUndataDir"/>
               </item>
              </layout>
             </item>
             <item row="2" column="1" rowspan="2">
              <widget class="QPushButton" name="svmSelUndataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" rowspan="2">
              <layout class="QFormLayout" name="formLayout_13">
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>异常测试</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="svmUnTestDir"/>
               </item>
              </layout>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="svmSelUnTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>模型路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="svmModelDir">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="svmSelBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="trainSvmBtn">
             <property name="text">
              <string>训练SVM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ANN操作区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>字符路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann1DataDir"/>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="ann1SelDataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <layout class="QFormLayout" name="formLayout_14">
               <item row="0" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>测试路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann1TestDir"/>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="ann1SelTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QFormLayout" name="formLayout_15">
               <item row="0" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>异常数据</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann1UndataDir"/>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="ann1SelUndataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <layout class="QFormLayout" name="formLayout_16">
               <item row="0" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>异常测试</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann1UnTestDir"/>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="ann1SelUnTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>模型路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann1ModelDir">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="ann1SelBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="trainAnnBtn">
             <property name="text">
              <string>训练ANN</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>ANN操作区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>字符路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann2DataDir"/>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="ann2SelDataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <layout class="QFormLayout" name="formLayout_17">
               <item row="0" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>测试路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann2TestDir"/>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="ann2SelTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QFormLayout" name="formLayout_18">
               <item row="0" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>异常数据</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann2UndataDir"/>
               </item>
              </layout>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="ann2SelUndataBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <layout class="QFormLayout" name="formLayout_19">
               <item row="0" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>异常测试</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann2UnTestDir"/>
               </item>
              </layout>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="ann2SelUnTestBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <layout class="QFormLayout" name="formLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>模型路径</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ann2ModelDir">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="1" colspan="2">
              <widget class="QPushButton" name="ann2SelBtn">
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="trainAnnZhBtn">
             <property name="text">
              <string>训练中文 </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
